### ISSUE TITLE
How to disable type cheking for not ts files

### ISSUE THREAD

**Issue by max-mykhailenko** [microsoft/TypeScript-Sublime-Plugin#744]
Some of my projects use typescript, some not. But error messages I see across all project.

```
Type annotations can only be used in TypeScript files.

any
```

Will be better to check `tsconfig` file and only if it available — turn on plugin.
What do you think?

**Comment by orta** [comment]
Is this in a flow file? It might be a good idea to check for a `.flowconfig` and disable the plugin

**Comment by max-mykhailenko** [comment]
Yes, it's flow file with flow pragma on top

**Comment by orta** [comment]
Yep, then I think I'd take a PR which has a lookup to see if the JS file lives inside a folder with a .flowconfig and disables the type checking.

**Comment by DanielRosenwasser** [comment]
You can also set `enable_language_service_for_javascript: false` in your `Preferences.sublime-settings` instead. I think that's probably more ideal.
